### DISCUSSION TITLE
GRASP는 무엇인가요?

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#39] [General]
# GRASP는 무엇인가요?

스터디 주제 준비 [ default-method ] [ functional-interface ] [ grasp ] 면접 [ bepohuhemal-study-question ]

https://cyma-s.github.io/grasp/

**Comment by greeng00se** [comment]
체스 미션을 예시로 든 부분이 인상깊어요! 잘 읽고 갑니다.

**Reply by Cyma-s** [reply]
허브의 칭찬 ... 귀하네요 😎
